db-constraints/src/main/java/org/evomaster/constraint/EnumConstraint.java

```diff
@@ -0,0 +1,24 @@
+package org.evomaster.constraint;
+
+import java.util.List;
+
+public class EnumConstraint extends TableConstraint {
+
+    private final List<String> valuesAsStrings;
+
+    private final String columnName;
+
+    public EnumConstraint(String tableName, String columnName, List<String> valuesAsStrings) {
+        super(tableName);
+        this.columnName = columnName;
+        this.valuesAsStrings = valuesAsStrings;
+    }
```

Comment:
in this class (and the others), if an input X must not be null, could write it as 
`this.x = Objects.requireNonNull(x);`

Reply:
Yes. I missed that. I added the checks on the missing classes in packages org.evomaster.dbconstraint, org.evomaster.client.java.controller.internal.db.constraint and org.evomaster.dbconstraint.ast